resources/views/components/badge.blade.php

```diff
@@ -0,0 +1,23 @@
+<span {{ $attributes->class($badgeClasses) }}>
+
+    @if ($icon)
```

Comment:
I would recommend adding the icon on the right as well.

Reply:
Added right-icon